src/business_logic/execution/objects.rs

```diff
@@ -59,17 +59,36 @@ impl TransactionExecutionContext {
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub(crate) struct OrderedL2ToL1Message {
-    pub(crate) _order: usize,
-    pub(crate) _to_address: usize,
-    pub(crate) _payload: Vec<Felt>,
+    pub(crate) order: usize,
+    pub(crate) to_address: usize,
+    pub(crate) payload: Vec<Felt>,
 }
 
 impl OrderedL2ToL1Message {
-    pub fn new(_order: usize, _to_address: usize, _payload: Vec<Felt>) -> Self {
+    pub fn new(order: usize, to_address: usize, payload: Vec<Felt>) -> Self {
```

Comment:
We are using u64 for addresses, is there any reason why you used usize?

Reply:
i was not sure which were we using, usize seemed accurate to me because it turns into u64 or u32 according to what is given as parameter, i will fix it to use u64 nevertheless